packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -309,6 +309,7 @@ $search-transition-timing: 95ms;
   // masthead profile menu
   .#{$prefix}--overflow-menu-options {
     background-color: $ui-background;
+    position: fixed;
```

Comment:
Approved. @kennylam , with this are the hooks you added for the React one still needed? I think I recall you were adding the position fixed dynamically or something?

Reply:
@jeffchew Yes, this is the first thing i tried on the `react` package, thinking it should be this simple. But for some reason the React version inlines all the styles on the component, making this impossible. Only way to override that one is to either go `!important` or rewrite the actual property, which is what I did.

https://github.com/carbon-design-system/carbon/blob/0719ef89fb25fc34655abed891257d85e7ff1264/packages/react/src/internal/FloatingMenu.js#L350-L358

Reply:
Got it, I guess this is React being React.